ssoPlugin/control/settings/index.html

```diff
@@ -0,0 +1,194 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<!-- The BuildFire JS library is required. -->
+	<script src="../../../../scripts/buildfire.min.js"></script>
+	<style type="text/css">
```

Comment:
Cool, glad you think so. Yes, assets specifically can't be shared between plugins (and even between control and widget side of the plugins) by just, like, sharing the file references. There is a Webpack-based template for new BuildFire plugins you can generate with the BuildFire CLI tool, which has you actually packing the plugin data before uploading; presumably that would let us share assets more, but I shoved that off to #8 as something to explore in the future.

Reply:
Oh but to sort of answer your first question: the aim there would be to share assets between plugins / parts of the plugin for ease of development. For performance reasons it would almost certainly be more efficient to inline this amount of CSS rather than create another HTTP request, although for practical purposes the advantage is probably negligible.